### PR TITLE
Update for changes to access/embargo in cocina models.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Cocina models now enforces valid access/embargo values.

## How was this change tested?
Unit

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

File: spec/jobs/ingest_job_spec.rb [hunk 1]
```diff
@@ -65,7 +67,8 @@
         shelve: true
       },
       access: {
-        access: 'citation-only'
+        access: 'stanford',
+        download: 'stanford'
```
Comment (ndushay) [inline spec/jobs/ingest_job_spec.rb:71]: would we in any way be better served keeping this citation only or the new equivalent and making a separate test for stanford access?

File: spec/jobs/update_job_spec.rb [hunk 2]
```diff
@@ -67,7 +69,8 @@
         shelve: true
       },
       access: {
-        access: 'citation-only'
+        access: 'stanford',
+        download: 'stanford'
```
Comment (ndushay) [inline spec/jobs/update_job_spec.rb:73]: same question here

File: spec/requests/create_resource_spec.rb [hunk 3]
```diff
@@ -110,8 +111,8 @@
                       "shelve":true
                     },
                     "access": {
-                      "access":"citation-only",
-                      "download":"none"
+                      "access":"stanford",
+                      "download":"stanford"
```
Comment (ndushay) [inline spec/requests/create_resource_spec.rb:115]: and here

File: spec/requests/update_resource_spec.rb [hunk 4]
```diff
@@ -66,8 +67,8 @@
                     "shelve":true
                   },
                   "access": {
-                    "access":"citation-only",
-                    "download":"none"
+                    "access":"stanford",
+                    "download":"stanford"
```
Comment (ndushay) [inline spec/requests/update_resource_spec.rb:71]: and here.

**Review by ndushay — COMMENTED** [review]
wondering if changing the "citation-only" examples to "stanford" permissions in all places is missing an opportunity to try some diff settings ...

Otherwise looks ok.

File: spec/jobs/ingest_job_spec.rb (hunk 1 above)
Comment (justinlittman) [reply spec/jobs/ingest_job_spec.rb:71]: `citation-only` isn't a valid value for a file. The actual values don't matter, as long as they're valid.

File: spec/jobs/update_job_spec.rb (hunk 2 above)
Comment (justinlittman) [reply spec/jobs/update_job_spec.rb:73]: Same answer as above.

File: spec/requests/create_resource_spec.rb (hunk 3 above)
Comment (justinlittman) [reply spec/requests/create_resource_spec.rb:115]: Same. `citation-only` isn't valid.

File: spec/requests/update_resource_spec.rb (hunk 4 above)
Comment (justinlittman) [reply spec/requests/update_resource_spec.rb:71]: Guess what my answer is going to be?

**Review by jcoyne — APPROVED** [review]
looks fine to me

**Pull request merged**